get-started/index.md

```diff
@@ -59,10 +62,320 @@ Click "Install" and wait for the installation to complete.
+to learn how to configure your application without the Cryostat Agent.
+
+The Cryostat Agent JAR must be available to your application JVM. The JAR asset can be downloaded [directly from upstream](https://github.com/cryostatio/cryostat-agent/releases),
+or you may use the following snippet in your `pom.xml` to streamline this.
+
+```xml
+<project>
+  ...
+  <repositories>
+    <repository>
+      <id>github</id>
+      <url>https://maven.pkg.github.com/cryostatio/cryostat-agent</url>
+    </repository>
+  </repositories>
+  ...
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <version>3.3.0</version>
+        <executions>
+          <execution>
+            <phase>prepare-package</phase>
+            <goals>
+              <goal>copy</goal>
+            </goals>
+            <configuration>
+              <artifactItems>
+                <artifactItem>
+                  <groupId>io.cryostat</groupId>
+                  <artifactId>cryostat-agent</artifactId>
+                  <version>0.2.0</version>
+                </artifactItem>
+              </artifactItems>
+              <stripVersion>true</stripVersion>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
+    ...
+  </build>
+  ...
+</project>
+```
+
+**Note**: You may be required to [authenticate to the GitHub Maven Packages registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-apache-maven-registry#installing-a-package) in order to pull this JAR.
+
+The next time we build our application, the Cryostat Agent JAR will be located at `target/dependency/cryostat-agent.jar`. Then we can update our Dockerfile:
+
+```Dockerfile
+...
+COPY target/dependency/cryostat-agent.jar /deployments/app/
+...
+# We are using a framework where the JAVA_OPTS environment variable can be used to pass JVM flags
+ENV JAVA_OPTS="-javaagent:/deployments/app/cryostat-agent.jar"
+```
+
+Next we must rebuild our container image. This is specific to your application but will likely look something like `docker build -t docker.io/myorg/myapp:latest -f src/main/docker/Dockerfile .`.
+Push that updated image or otherwise get it updated in your Kubernetes registry, then modify your application `Deployment` to supply JVM system properties or environment variables configuring
+the Cryostat Agent:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+...
+spec:
+  ...
+  template:
+    ...
+    spec:
+      containers:
+        - name: sample-app
+          image: docker.io/myorg/myapp:latest
+          env:
+            - name: CRYOSTAT_AGENT_APP_NAME
+              value: "myapp"
+              # Replace this with the Kubernetes DNS record
+              # for the Cryostat Service
+            - name: CRYOSTAT_AGENT_BASEURI
```

Comment:
from these details- I see that baseuri and callback is required. where do I get these values?

Reply:
The comment above:

>              # Replace this with the Kubernetes DNS record
>              # for the Cryostat Service

is referring to the server baseuri. I can make this more detailed if needed, but it's [basically this standard thing](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/) that just depends on what the user named their Cryostat instance and what namespace they installed it into.

The agent callback value is already set up in this example to be inferred from the Kubernetes pod IP for the application deployment automatically.